scripts/gui/CourseDisplay.lua

```diff
@@ -259,22 +259,30 @@ function SimpleCourseDisplay:updateChangesBetween(firstIx, secondIx)
 end
 
 --- Changes the visibility of the course.
-function SimpleCourseDisplay:updateVisibility(visible, onlyStartStopVisible)
+function SimpleCourseDisplay:updateVisibility(visible, onlyStartStopVisible, currentWaypoint)
 	if self.course then
 		local numWp = self.course:getNumberOfWaypoints()
 		for j = 1, numWp do
-			if self.signs[j] then 
+			if self.signs[j] then
 				self.signs[j]:setVisible(visible)
-				if not self.signs[j]:isNormalSign() or j == numWp - 1 then 
-					self.signs[j]:setVisible(visible or onlyStartStopVisible)
-				end	
+				if not self.signs[j]:isNormalSign() or j == numWp - 1 then

```

Comment:
There are alot of contants here: ``local lowerBound = currentWaypoint - 5
						local upperBound = currentWaypoint + 20``
Might be nice to have a comment explaning the usage and also move these to global constants of the class ``SimpleCourseDisplay``

Reply:
Moving them into a constant is something i've planed, but intentionally didn't do for now since my goal is to make there constants more configurable and I don't want to move them twice. 
So for now I would leave them there but add a TODO with the mention to move them to a global variable.

Reply:
@tn4799 That's okay for now